### PR TITLE
chore: dependencies and release

### LINKED ISSUE #74: New release to include last supported Linux versions
The infra agent now supports:
* Amazon Linux 2022 (Public Preview, GA expected by ~July)
* RHEL 9 (GA)
* Ubuntu 22.04 (GA)

### PR DESCRIPTION
**Pull request by sigilioso**
closes #74

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,4 +41,4 @@ install: compile
 include $(CURDIR)/build/ci.mk
 include $(CURDIR)/build/release.mk
 
-.PHONY: all build clean validate compile test integration-test install
\ No newline at end of file
+.PHONY: all build clean validate compile test integration-test install
```
Comment (carlossscastro) [inline Makefile:44]: Shouldn't we remove validate?

**Review by carlossscastro — CHANGES_REQUESTED** [review]
We could also bump dependencies: 
https://github.com/newrelic/nri-apache/blob/master/go.mod

At least our SDK and and testify have newer version

File: Makefile (hunk 1 above)
Comment (sigilioso) [reply Makefile]: Absolutely, thanks!  Deleted: [fbe9aa5](https://github.com/newrelic/nri-apache/pull/76/commits/fbe9aa539fb57009e99bbf26e34fb2478de9af03)

**Review by gsanchezgavier — APPROVED** [review]
thanks , i would just re add the EOL on the files that are marked as missing

**Comment by sigilioso** [comment]
> We could also bump dependencies: https://github.com/newrelic/nri-apache/blob/master/go.mod
> 
> At least our SDK and and testify have newer version

Updated here [af692ce](https://github.com/newrelic/nri-apache/pull/76/commits/af692ce28c85729b27e7dd4c1ee62a1786fa8676), thanks!

**Review by carlossscastro — APPROVED** [review]
Perfect. Thanks!

**Pull request merged**